examples/event_handling/pick_event_demo.py

```diff
@@ -100,8 +102,11 @@ def onpick1(event):
             print('onpick1 text:', text.get_text())
 
     fig.canvas.mpl_connect('pick_event', onpick1)
+    plt.show()
```

Comment:
It used to just show all four figure before. Is there some reason we need `plt.show` at the end of every function?

Reply:
Since we are calling the functions one by one, hence we need the ```plt.show```.
The other option would be to change the functions by passing the fig handles and making figures, which is too complex for a simple example.

Awaiting your comments though!

Reply:
I do not understand; separate functions does not imply that there cannot be just one `plt.show` at the end.

Reply:
okay, will change it to that!